docs/source/python_api/mlflow.metrics.rst

```diff
@@ -98,26 +98,27 @@ Users create their own :py:class:`EvaluationMetric <mlflow.metrics.EvaluationMet
 
 .. autofunction:: mlflow.metrics.make_metric
 
-We provide the following pre-canned "intelligent" :py:class:`EvaluationMetric <mlflow.metrics.EvaluationMetric>` for evaluating text models. These metrics use an LLM to evaluate the quality of a model's output text. Note that your use of a third party LLM service (e.g., OpenAI) for evaluation may be subject to and governed by the LLM service's terms of use. The following factory functions help you customize the intelligent metric to your use case.
-
-.. autofunction:: mlflow.metrics.answer_similarity
-
-.. autofunction:: mlflow.metrics.answer_correctness
+.. automodule:: mlflow.metrics
+    :members:
+    :undoc-members:
+    :show-inheritance:
+    :exclude-members: MetricValue, EvaluationMetric, make_metric, EvaluationExample, ari_grade_level, flesch_kincaid_grade_level, rouge1, rouge2, rougeL, rougeLsum, toxicity, answer_similarity, answer_correctness, faithfulness, answer_relevance, mae, mape, max_error, mse, rmse, r2_score, precision_score, recall_score, f1_score, token_count, latency
 
-.. autofunction:: mlflow.metrics.faithfulness
+Generative AI Metrics
+---------------------
```

Comment:
@prithvikannan @annzhang-db @sunishsheth2009 let me know what you  think about this naming. Happy to adjust for consistency with the rest of our docs updates.

Reply:
I think this makes sense considering users will be accessing these metrics through `mlflow.metrics.genai.metric_name` and it is consistent with that experience - @sunishsheth2009 @prithvikannan thoughts?

Reply:
lgtm!